decoimpact/business/application.py

```diff
@@ -71,37 +71,53 @@ def run(self, input_path: Path):
             self._logger.log_info(f"Input file version: {str_input_version}")
 
             # check version:
-            error_msg = (
-                f"Application version {self.APPLICATION_VERSION} is older"
-                " than version from input file {str_input_version}"
-            )
-            warning_msg = (
+            message = (
                 f"Application version {self.APPLICATION_VERSION} is older"
                 " than version from input file {str_input_version}"
             )
             # major version (app) should be equal or larger then input version --> error
             if self.APPLICATION_VERSION_PARTS[0] < model_data.version[0]:
-                self._logger.log_error(error_msg)
+                self._logger.log_error(message)
             # minor version (app) should be equal or larger then input version --> warn
             elif self.APPLICATION_VERSION_PARTS[1] < model_data.version[1]:
-                self._logger.log_warning(warning_msg)
+                self._logger.log_warning(message)
 
             # build model
-            model = self._model_builder.build_model(model_data)
-
-            # run model
-            _ModelRunner.run_model(model, self._logger)
-
-            # write output file
-            if model.status == _ModelStatus.FINALIZED:
-                settings = OutputFileSettings(
-                    self.APPLICATION_NAME, self.APPLICATION_VERSION
-                )
-                settings.variables_to_save = model_data.output_variables
-
-                self._da_layer.write_output_file(
-                    model.output_dataset, model_data.output_path, settings
+            for dataset in model_data.datasets:
+                input_files = self._da_layer.retrieve_partitioned_file_names(
```

Comment:
I would rename this function to something like 'retrieve_file_names', since not only partitioned files are read (this can be confusing if you are not using partitioned file names). Maybe add a comment to explain that in case of partitioned files, multiple files are read.

Reply:
changed name
added comment